### PR TITLE
Refine Query Filters for Recipients of Notification

### PR DESCRIPTION
**Pull request by kenlewis-sfdo**
[W-10164853](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000OHoCVYA1/view)

# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# Features Intended for Future Release

# Features for Elevate Customers

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: force-app/main/default/classes/UserSelector.cls [hunk 1]
```diff
@@ -35,10 +35,13 @@
...
 public with sharing class UserSelector {
 
-   public List<User> getUsers() {
+   public List<User> getActiveStandardUsers() {
       return [
           SELECT Id
-          FROM User LIMIT 10000
+          FROM User
+          WHERE IsActive = TRUE
+          AND UserType = 'Standard'
```
Comment (force2b) [inline force-app/main/default/classes/UserSelector.cls:43]: Should this alert go to all `Standard` users or just those with some level of system admin permission, such as those with CustomizeApplication?

Comment (kenlewis-sfdo) [reply force-app/main/default/classes/UserSelector.cls:43]: The [WI ](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07AH000000RIKsYAO/view)calls for delivery to all Users, not just Admins, so I don't think the check for an Admin perm is necessary.

**Comment by lparrott** [comment]
@kenlewis-sfdo Changes look good, just remember to also uncomment the `new InstallationNotifications` line in `STG_InstallScript`!

**Review by lparrott — APPROVED** [review]
The new test looks great!

**Pull request merged**